landing-page/content/common/how-to-release.md

```diff
@@ -21,6 +21,18 @@ disableSidebar: true
  - limitations under the License.
  -->
 
+## Introduction
+
+This page walks you through the release process of the Iceberg project. [Here](https://www.apache.org/legal/release-policy.html) you can read about the release process in general for an Apache project.
+
+Decisions about releases are made by three groups:
+
+* Release Manager: Does the work of creating the release, signing it, counting [votes](/how-to-release/#voting), announcing the release and so on. Requires the assistance of a committer for some steps.
+* The community: Performs the discussion of whether it is the right time to create a release and what that release should contain. The community can also cast non-binding votes on the release.
+* PMC: Gives binding votes on the release.
```

Comment:
nit: a bit awkward to only put binding vote as a bullet point. Can we move that as a part of the last sentence?

Reply:
actually, looks like we also explain binding and non-binding in L221, should we just remove it here?